sqld/src/main.rs

```diff
@@ -336,6 +336,10 @@ fn enable_libsql_logging() {
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    if std::env::var("RUST_LOG").is_err() {
```

Comment:
I couldn't get RUST_LOG to work locally...so I am doing it this way and it seems to work better. Idk why the previous setup was not working.